src/components/OnboardingMenu/OnboardingMenu.scss

```diff
@@ -0,0 +1,108 @@
+@use '../variables.scss';
+@use '@gravity-ui/uikit/styles/mixins';
+
+$block: '.#{variables.$ns}guide';
+
+$layer-popup: 1000;
+
+#{$block} {
+    max-width: 280px;
+
+    background: none;
+    width: fit-content;
+
+    &__content {
+        box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.04);
+        display: flex;
+        flex-direction: column;
+        overflow: hidden;
+
+        box-sizing: border-box;
+        width: 210px;
+
+        background-color: var(--g-color-base-brand);
+        border-radius: 16px;
+
+        &_expand {
+            width: 280px;
+            border-radius: 20px;
+            height: fit-content;
+            min-height: 300px;
+        }
+    }
+
+    &__title-text {
+        color: var(--g-color-text-brand-contrast);
+        @include mixins.text-subheader-2();
+
+        overflow: hidden;
+        text-overflow: ellipsis;
+        text-wrap: nowrap;
+
+        &_expand {
+            @include mixins.text-subheader-3();
+        }
```

Comment:
It's better to put color and font styles on the button, thus you don't need to duplicate them on text and icon elements